HCSStarRatingView/HCSStarRatingView.m

```diff
@@ -62,6 +62,12 @@ - (void)_customInit {
     _value = 0;
     _spacing = 5.f;
     _continuous = YES;
+    _starBorderWidth = 1.0f;
+    _starBorderColor = self.tintColor;
```

Comment:
This shouldn't be set here, because if the tint color changes and we haven't set a custom border color, this would result in different colors for existing apps (which means we're braking API behaviour).

My suggestion is that we have a getter for `starBorderColor` that returns `_starBorderColor` if it's not `nil`, and `self.tintColor` otherwise.

This way if a developer already using this control updates the pod/code and doesn't care/know about custom border colors, his/her code won't break when (s)he tries to change the tint color.

What do you think?